### ISSUE TITLE
[Bug]: grcpio VersionConflict

### ISSUE THREAD

**Issue by shiyu22** [towhee-io/towhee#1913]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

When I start triton server, and run the following code
```python
res = towhee.dc['path'](['https://github.com/towhee-io/towhee/blob/main/towhee_logo.png?raw=true']) \
    .remote['path', 'vec'](url='172.16.70.4:8001', mode='infer', protocol='grpc') \
    .ann_insert.milvus['vec','mr'](uri='tcp://172.16.70.4:19530/triton')
```
It will show the VersionConflict error:
```bash
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/chenshiyu/workspace/git/towhee/towhee/hparam/hyperparameter.py", line 200, in __call__
    return self._func(*args, **kws)
  File "/Users/chenshiyu/workspace/git/towhee/towhee/functional/data_collection.py", line 143, in wrapper
    return self.map(op)
  File "/Users/chenshiyu/workspace/git/towhee/towhee/functional/data_collection.py", line 587, in map
    arg[0].__check_init__()
  File "/Users/chenshiyu/workspace/git/towhee/towhee/engine/factory.py", line 94, in __check_init__
    self._op = op(self._name,
  File "/Users/chenshiyu/workspace/git/towhee/towhee/engine/factory.py", line 61, in op
    return loader.load_operator(operator_src, arg, kwargs, tag)
  File "/Users/chenshiyu/workspace/git/towhee/towhee/engine/operator_loader.py", line 162, in load_operator
    op = factory(function, arg, kws, tag)
  File "/Users/chenshiyu/workspace/git/towhee/towhee/engine/operator_loader.py", line 141, in load_operator_from_cache
    return self.load_operator_from_path(path, arg, kws)
  File "/Users/chenshiyu/workspace/git/towhee/towhee/engine/operator_loader.py", line 101, in load_operator_from_path
    pkg_resources.require(req)
  File "/Users/chenshiyu/miniconda3/envs/shiyu/lib/python3.9/site-packages/pkg_resources/__init__.py", line 897, in require
    needed = self.resolve(parse_requirements(requirements))
  File "/Users/chenshiyu/miniconda3/envs/shiyu/lib/python3.9/site-packages/pkg_resources/__init__.py", line 788, in resolve
    raise VersionConflict(dist, req).with_context(dependent_req)
pkg_resources.ContextualVersionConflict: (grpcio 1.41.0 (/Users/chenshiyu/miniconda3/envs/shiyu/lib/python3.9/site-packages), Requirement.parse('grpcio>=1.47.0'), {'pymilvus'})
```
And when I reinstall grpcio>=1.47.0 it will show that tritonclient [requires grpcio==1.41.0](https://github.com/triton-inference-server/client/blob/main/src/python/library/requirements/requirements_grpc.txt#L34).

### Environment

```markdown
- Towhee version(e.g. v0.1.3 or 8b23a93):
- OS(Ubuntu or CentOS):
- CPU/Memory:
- GPU:
- Others:
```

**Comment by shiyu22** [comment]
There is a [PR](https://github.com/triton-inference-server/client/pull/92) to solve the VersionConflict, maybe we can use it.

**Comment by afj88** [comment]
Recently I've tried to use tritonclient on my mac M1 and I encountered this error

`File "/.../lmv1_triton_inference_client.py", line 7, in <module>
    import tritonclient.grpc as tritongrpcclient  # type: ignore
  File "/.../venv/lib/python3.10/site-packages/tritonclient/grpc/__init__.py", line 29, in <module>
    import grpc
  File "/.../venv/lib/python3.10/site-packages/grpc/__init__.py", line 22, in <module>
    from grpc import _compression
  File "/.../venv/lib/python3.10/site-packages/grpc/_compression.py", line 15, in <module>
    from grpc._cython import cygrpc
ImportError: dlopen(/.../venv/lib/python3.10/site-packages/grpc/_cython/cygrpc.cpython-310-darwin.so, 0x0002): tried: '/.../venv/lib/python3.10/site-packages/grpc/_cython/cygrpc.cpython-310-darwin.so' (mach-o file, but is an incompatible architecture (have (x86_64), need (arm64e)))`

With some investigation I discovered that this issue has been resolved in grpcio v1.47 as described in this issue
[grpc](https://github.com/grpc/grpc/issues/28387)

Could you please think about re-opening this issue?
